packages/react-native/src/Authenticator/common/DefaultContent/DefaultContent.tsx

```diff
@@ -44,12 +44,14 @@ export default function DefaultContent<
 
   return (
     <>
-      {headerText ? (
-        <Header style={themedStyles.header}>{headerText}</Header>
-      ) : (
-        <Header />
-      )}
-      {body ? body : null}
+      <Header style={themedStyles.header}>{headerText}</Header>
+      {body ? (
```

Comment:
Just wondering when `body` would be a `string`?

Reply:
When we just want to display a text, VerifyUSer screen is an example of that.